src/lib/autocomplete/autocomplete-trigger.ts

```diff
@@ -48,6 +56,22 @@ export const AUTOCOMPLETE_OPTION_HEIGHT = 48;
 /** The total height of the autocomplete panel. */
 export const AUTOCOMPLETE_PANEL_HEIGHT = 256;
 
+/** Injection token that determines the scroll handling while the autocomplete panel is open. */
+export const MD_AUTOCOMPLETE_SCROLL_STRATEGY =
+    new InjectionToken<() => ScrollStrategy>('md-autocomplete-scroll-strategy');
+
+/** @docs-private */
+export function MD_AUTOCOMPLETE_SCROLL_STRATEGY_PROVIDER_FACTORY(overlay: Overlay) {
+  return () => overlay.scrollStrategies.reposition();
```

Comment:
wouldn't better to handle to case of IOS too (related to https://github.com/angular/material2/issues/5187)
```ts
return () => platform.IOS ? overlay.scrollStrategies.block() : overlay.scrollStrategies.reposition();
```